.mailmap

```diff
@@ -365,6 +365,8 @@ Ayush Malik <ayushmalik779@gmail.com> Ayush-Malik <ayushmalik779@gmail.com>
 Ayush Shridhar <ayush.shridhar1999@gmail.com>
 Ayushman Koul <ayushmankoul4570@gmail.com>
 B McG <bmcg0890@gmail.com>
+Baiyuan Qiu <1061688677@qq.com> <qby1061688677@gmail.com>
+Baiyuan Qiu <51898470+bugmaker2@users.noreply.github.com>
```

Comment:
You should not add multiple lines to the .mailmap file like this:
https://github.com/sympy/sympy/blob/a775270e14bdedc4a39b4fb9c6e4e6e398be896f/.mailmap#L125-L148
@sylee957 can you check for this when merging PRs?